### DISCUSSION TITLE
argocd cluster add --namespace: Is there a way to use wildcard with --namespace?

### DISCUSSION THREAD

**Discussion by pon-raj** [argoproj/argo-cd#11634] [Q&A]
Hi,
When I use argocd cluster with --namespace option, it does not seem to be accpting wildcard. Instead of keeping an array of namespaces, I tried, for an example, *-prod, a subset of namespaces that I want to manage that namespace names end with "-prod".

Is there a way to use wildcard with --namespace?

Any advice, please.

**Comment by agaudreault** [comment]
I think this is the feature request with https://github.com/argoproj/argo-cd/issues/10054
